apps/backend/src/parse/parse.controller.ts

```diff
@@ -69,6 +69,23 @@ export class ParseController {
     @Body() body: { program: Ministries; fileType: FileTypes },
     @UploadedFile() file: Express.Multer.File
   ) {
+    const fileSplit = file.filename.split('.');
+    const fileExtension = fileSplit[fileSplit.length - 1];
+    if (
+      fileSplit.length < 2 ||
```

Comment:
This looks good... I know that multer has some options that can be configured for validation as well - I think mimetype is one of them... although I am not sure re: DAT/JSON but I think you can specify 🤷‍♀️